draft-savage-ppm-3phm-mpc.md

```diff
@@ -376,8 +376,8 @@ replicated secret sharing of the product of `x` and `y`.
 
 # Validation Protocol {#validation}
 
-The basic multiplication protocol in {{multiplication}} only offers "semi-honest
-security". It is "secure up to an additive attack". Validating multiplications
+The basic multiplication protocol in {{multiplication}} only offers semi-honest
+security. It is "secure up to an additive attack". Validating multiplications
```

Comment:
@martinthomson had a comment in the original Google doc saying that we should probably define the things in scare quotes up in the definitions section... But I felt like these terms were already defined in the text immediately following the introduction of these terms. @martinthomson - what do you think we should do? Should we add definitions for these things or is the current text OK?